modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -31,6 +31,11 @@ def submit
           ews = create_ews(params[:id])
           tracked_item_ids&.each { |id| ews.tracked_items << id }
           ews.save
+
+          ClaimsApi::Logger.log('EWS_controller',
```

Comment:
This feels redundant. Sidekiq already logs the starting of jobs. Additionally, `EvidenceWaiverBuilderJob` [is already logging](https://github.com/department-of-veterans-affairs/vets-api/blob/a14194c8355d763e3353f57a94c20bd1d5d3b128/modules/claims_api/app/sidekiq/claims_api/evidence_waiver_builder_job.rb#L32) when it fails to upload, but also doesn't have any real detail like the ews or claim IDs. Let's add those details there and leave this out.